eclipse/sw360
# feat(ui):Rename and Re-arrange the Tabs under Projects section

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

>  Renamed and re-arranged the projects tab.
> * Which issue is this pull request belonging to and how is it solving it? (#1108 )
> * Did you add or update any new dependencies that are required for your change? No

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
> How should these changes be tested by the reviewer? 
   - Open sw360, click on the Projects tab and open a project and validate the different tabs under project section.
> Have you implemented any additional tests? No

### Checklist
Must:

Signed-off-by: Ravindra Kumar kumar.ravindra@siemens.com
